Secrets plugin fails to compile

Looks like `secrets/api.go` got left out of 0ac8926da3e6d36ebc1bd8319449c27cc6901a81:

```console
$ pwd
/go/src/github.com/docker/go-plugins-helpers/secrets
$ go build .
# github.com/docker/go-plugins-helpers/secrets
./api.go:72:33: cannot use res.Err (type string) as type bool in argument to sdk.EncodeResponse
```

PR incoming to fix this...

--- a/secrets/api.go
+++ b/secrets/api.go
@@ -1,8 +1,9 @@
 package secrets
 
 import (
-	"github.com/docker/go-plugins-helpers/sdk"
 	"net/http"
+
+	"github.com/docker/go-plugins-helpers/sdk"
 )
 
 const (
@@ -69,6 +70,6 @@ func (h *Handler) initMux() {
 			return
 		}
 		res := h.driver.Get(req)
-		sdk.EncodeResponse(w, res, res.Err)
+		sdk.EncodeResponse(w, res, res.Err != "")
 	})
 }

